external/fixedWidth/pom.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this intentional and do we want to remove this from other pom files as well?

Reply:
Yeah, most didn't have it. It was either we add it to the rest or remove it from the ones that had it. It seemed extraneous to me, so I removed it where I found it. If you think it makes more sense to have it, happy to add it to all.

Reply:
So I clicked a bit around the net and half the projects I saw had them, half didn't. I particularly don't care, but I think some validators/plugins require them, but we don't have that problem.

Would reopen this issue when a problem occurs.